core/templates/services/user.service.ts

```diff
@@ -56,7 +56,7 @@ export class UserService {
         this.urlInterpolationService.getStaticImageUrl(
           AppConstants.DEFAULT_PROFILE_IMAGE_PATH));
       return this.getUserInfoAsync().then(
-        (userInfo) => {
+        async(userInfo) => {
```

Comment:
Hmm, should we put space after `async`?

Reply:
I don't think we can put a space here, for anonymous functions, if you add a space, the eslint gives an error.

Reply:
Thanks @srijanreddy98.